### PR TITLE
Bump python to 3.7.7

### PR DESCRIPTION
**Pull request by RDIL**
Still works

### CODE REVIEW

**Comment by JesterOrNot** [comment]
Fails with
image

**Comment by jankeromnes** [comment]
The screenshot only shows which command failed. This seems to be the actual error message:

```
              Installing Python-2.7.17...
              Installed Python-2.7.17 to /home/gitpod/.pyenv/versions/2.7.17
              
              python-build: definition not found: 3.7.7
              
              See all available versions with `pyenv install --list'.
              
              If the version you need is missing, try upgrading pyenv:
              
                cd /home/gitpod/.pyenv/plugins/python-build/../.. && git pull && cd -
```

**Comment by RDIL** [comment]
Yeah, pyenv doesn't have a database entry for it yet.

**Comment by JesterOrNot** [comment]
What about bumping to `3.8.0`

**Comment by RDIL** [comment]
Sean, **tensorflow doesn't work on python 3.8**.

**Comment by JesterOrNot** [comment]
Right, my bad

**Comment by JesterOrNot** [comment]
blocked by https://github.com/pyenv/pyenv/issues/1552

File: full/Dockerfile [hunk 1]
```diff
@@ -206,8 +206,8 @@ RUN curl -fsSL https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-ins
         echo 'eval "$(pyenv init -)"'; \
         echo 'eval "$(pyenv virtualenv-init -)"'; } >> /home/gitpod/.bashrc.d/60-python \
     && pyenv install 2.7.17 \
```
Comment (JesterOrNot) [inline full/Dockerfile]: Add this to get 3.7.7 to work
```Dockerfile
&& pyenv update \
```
Our `pyenv` version is out of date

Comment (JesterOrNot) [reply full/Dockerfile]: Thanks!

**Comment by JesterOrNot** [comment]
It works!
image

**Pull request merged**